lib/server/services/log/LogService.js

```diff
@@ -85,6 +85,20 @@ class LogService {
         return { count, logIds };
     }
 
+    /**
+     * Retrieves a list of replies to a log based off the given logId
+     * @param {number} logId The id of the log of which the child logs are retrieved
+     * @return {Promise<{childLogs: Log[]}>} A promise that resolves with an object containing
+     * an array of the child logs for the given parent log
```

Comment:
Don't forget to update the type of return, it's not a object with `childLogs` anymore 🙂 

Also, I think that you can simplify a bit the text, something like `Resolves with the replies logs` would be clear enough I think.

Reply:
done